nucypher/blockchain/eth/actors.py

```diff
@@ -248,39 +258,65 @@ def deploy_user_escrow(self, allocation_registry: AllocationRegistry):
-                                 ) -> Tuple[dict, dict]:
-        """
-        Musketeers, if you will; Deploy the "big three" contracts to the blockchain.
-        """
-
-        token_txs, token_deployer = self.deploy_contract(contract_name='NuCypherToken')
-        staking_txs, staking_deployer = self.deploy_contract(contract_name='StakingEscrow', plaintext_secret=staker_secret)
-        policy_txs, policy_deployer = self.deploy_contract(contract_name='PolicyManager', plaintext_secret=policy_secret)
-        adjudicator_txs, adjudicator_deployer = self.deploy_contract(contract_name='Adjudicator', plaintext_secret=adjudicator_secret)
-        user_escrow_proxy_txs, user_escrow_proxy_deployer = self.deploy_contract(contract_name='UserEscrowProxy', plaintext_secret=user_escrow_proxy_secret)
-
-        deployers = (token_deployer,
-                     staking_deployer,
-                     policy_deployer,
-                     adjudicator_deployer,
-                     user_escrow_proxy_deployer,
-                     )
-
-        txhashes = {
-            NucypherTokenDeployer.contract_name: token_txs,
-            StakingEscrowDeployer.contract_name: staking_txs,
-            PolicyManagerDeployer.contract_name: policy_txs,
-            AdjudicatorDeployer.contract_name: adjudicator_txs,
-            UserEscrowProxyDeployer.contract_name: user_escrow_proxy_txs,
-        }
-
-        deployers = {deployer.contract_name: deployer for deployer in deployers}
-        return txhashes, deployers
+    def deploy_network_contracts(self, secrets: 'Secrets', interactive: bool = True) -> dict:
+
+        deployment_receipts = dict()
+        gas_limit = 8_000_000  # TODO: Gas management
+
+        # NuCypherToken
+        token_receipts, token_deployer = self.deploy_contract(contract_name=NucypherTokenDeployer.contract_name,
+                                                              gas_limit=gas_limit)
+        paint_contract_deployment(contract_name=NucypherTokenDeployer.contract_name,
+                                  receipts=token_receipts,
+                                  contract_address=token_deployer.contract_address)
+        deployment_receipts[NucypherTokenDeployer.contract_name] = token_receipts
+
+        if interactive:
+            click.pause(info="Press any key to continue")
+
+        # StakingEscrow
+        staking_receipts, staking_deployer = self.deploy_contract(contract_name=StakingEscrowDeployer.contract_name,
+                                                                  plaintext_secret=secrets.staker_secret,
+                                                                  gas_limit=gas_limit)
+        paint_contract_deployment(contract_name=StakingEscrowDeployer.contract_name,
+                                  receipts=staking_receipts,
+                                  contract_address=staking_deployer.contract_address)
+        deployment_receipts[StakingEscrowDeployer.contract_name] = staking_receipts
```

Comment:
Same comment as above - a reader has to assume that `StakingEscrowDeployer.contract_name`, but since this doesn't occur in a loop, the domain of the key space of this dict is difficult to grok.

Reply:
:heavy_check_mark: